docs/wiki/research/SSF.md

```diff
@@ -0,0 +1,57 @@
+# Single slot finality (SSF)
+
+## Roadmap tracker
+
+| Upgrade |   URGE    | Track |          Topic           |                                               Cross-references                                               |
+| :-----: | :-------: | :---: | :----------------------: | :----------------------------------------------------------------------------------------------------------: |
+|   SSF   | the Merge |   -   | Increase Validator count | intersection with: [MAX_EB](/docs/wiki/research/cl-upgrades.md), [ePBS](/docs/wiki/research/PBS/ePBS.md), ET |
+
+## What is SSF?
+[Single Slot Finality](/docs/eps/week10-research.md) is a concept within Ethereum, in the context of consensus mechanism, that addresses the inefficiencies associated with the time it takes to finalize blocks, proposing a significant raise in blocks validation efficiency and a drastic reduction of time-to-finality. 
+Instead of waiting for 2 epochs, blocks could get proposed and finalized in the same slot.
+
+## Context and Motivation
+Ethereum finalizes every 2 epochs, or every 64 slots. With each slot being 12 seconds long, finalization takes around 12.8 minutes, at the moment of writing.
```

Comment:
Before this sentence, there should be also an intro to what is finality/weak subjectivity. Something along:

```suggestion
Ethereum consensus layer implements Gasper protocol which includes Casper Friendly Finality Gadget. Casper FFG ensures that the network keeps producing blocks and accumulates validator attentions for each epoch. Finality is the ultimate state of PoS economic security and its change would require 2/3 of the validator set to be slashed. 
Beacon Chain achieves finality every 2 epochs, or every 64 slots. With each slot being 12 seconds long, finalization takes around 12.8 minutes, at the moment of writing.
```

Reply:
Maybe also add a reference to PoS docs explaining finality